source/coding-the-legislation/legislation_parameters.md

```diff
@@ -4,45 +4,46 @@
 
 The parameters are organized with in a [tree structure](https://en.wikipedia.org/wiki/Tree_structure).
 
->**Example**: `tax_on_salary.public_sector.rate` can be found in `parameters/tax_on_salary/public_sector/rate.yml`.
+> **Example**: `tax_on_salary.public_sector.rate` can be found in `parameters/tax_on_salary/public_sector/rate.yml` .
 
 Example of a `parameters` directory:
 * parameters
-  * tax_on_salary
-    * **tax_scale.yaml**
-    * public_sector
+  + tax_on_salary
+    - **tax_scale.yaml**
+    - public_sector
       * **rate.yaml**
-  * universal_income
-    * **minimum_age.yaml**
-    * **amount.yaml**
-
+  + universal_income
+    - **minimum_age.yaml**
+    - **amount.yaml**
 
 In this file structure:
- - `tax_on_salaries`, `tax_on_salary.public_sector`, `universal_income` are **nodes**;
- - `tax_on_salaries.tax_scale`, `tax_on_salary.public_sector.rate`,`universal_income.minimum_age`,`universal_income.amount` are **parameters** (or scales).
+ - `tax_on_salaries` , `tax_on_salary.public_sector` , `universal_income` are **nodes**;
+ - `tax_on_salaries.tax_scale` , `tax_on_salary.public_sector.rate` , `universal_income.minimum_age` , `universal_income.amount` are **parameters** (or scales).
```

Comment:
I'm not sure about these added spaces before commas. I understand that even though those are code items, they should be read as part of a sentence. Could you please roll this back? 🙂